pkg/strategy/xmaker/strategy.go

```diff
@@ -1027,7 +1064,7 @@ func (s *Strategy) tryArbitrage(ctx context.Context, quote *Quote, makerBalances
 			return false, nil
 		}
 
-		if qty.IsZero() {
+		if qty.IsZero() || s.makerMarket.IsDustQuantity(qty, sumPv.Price) {
```

Comment:
Does `qty.IsZero()` become redundant? Or `qty.IsZero()` is a faster route for most cases?